core/src/main/java/org/infinispan/configuration/global/GlobalSecurityConfigurationChildBuilder.java

```diff
@@ -12,7 +12,8 @@
     */
    GlobalAuthorizationConfigurationBuilder authorization();
    /**
-    * Defines the timeout in milliseconds for which to cache user access roles
+    * Defines the timeout in milliseconds for which to cache user access roles. A value of -1 means the entries
```

Comment:
I know this might be too late now.  But was there a reason we forced this to be milliseconds and didn't just do the new common method of passing a long and a TimeUnit?